.pre-commit-config.yaml

```diff
@@ -19,7 +19,7 @@ repos:
     rev: 20.8b1
     hooks:
       - id: black
-        exclude: docs/.*|tests/.*.fixture|.*.ge_store_backend_id
+        exclude: docs/.*|tests/.*.fixture|.*.ge_store_backend_id|DO_NOT_COMMIT/.*
```

Comment:
Is this something that can be added to your local configuration instead of to the library?

Reply:
deleted